packages/webapp/pages/squads/create.tsx

```diff
@@ -40,9 +55,28 @@ function CreatePost(): ReactElement {
   const { push, isReady: isRouteReady, query } = useRouter();
   const { squads, user, isAuthReady, isFetched } = useAuthContext();
   const [selected, setSelected] = useState(-1);
-  const activeSquads = squads?.filter(
-    (squad) => squad?.active && verifyPermission(squad, SourcePermissions.Post),
-  );
+  const activeSquads = useMemo(() => {
+    const filtered = squads
+      ?.filter(
+        (squaitem) =>
+          squaitem?.active &&
+          verifyPermission(squaitem, SourcePermissions.Post),
+      )
+      .sort((a, b) => sortAlphabetically(a.name, b.name));
+
+    if (!user) {
+      return filtered;
+    }
+
+    return [
+      {
+        ...generateDefaultSquad(user.username),
+        name: 'Create new Squad',
+        handle: null,
```

Comment:
Why do we do this here?

Reply:
Just so the dropdown would not have any handle displayed, as per Figma.